### PR TITLE
Remove Running Rewrite without build tool plugins

### PR DESCRIPTION
**Pull request by timtebeek**
We now recommend the Moderne CLI as an alternative to Maven and Gradle plugins.

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -1433,7 +1432,7 @@
       * [Ensure the GKE metadata server is enabled](reference/recipes/terraform/gcp/ensurethegkemetadataserverisenabled.md)
     * [Search](reference/recipes/terraform/search/README.md)
       * [Find Terraform resource](reference/recipes/terraform/search/findresource.md)
-  * [Core](reference/recipes/core-README.md)
+  * [Core](reference/recipes/core-readme.md)
```
Comment (mike-solomon) [inline SUMMARY.md:1435]: Why are we changing the capitalization here? Was it incorrect before?

File: running-recipes/running-recipes.md [hunk 2]
```diff
@@ -3,9 +3,8 @@
 * [Quickstart: Setting up your project and running recipes](/running-recipes/getting-started.md)
 * [Running Rewrite on a Gradle project without modifying the build](/running-recipes/running-rewrite-on-a-gradle-project-without-modifying-the-build.md)
 * [Running Rewrite on a Maven project without modifying the build](/running-recipes/running-rewrite-on-a-maven-project-without-modifying-the-build.md)
-* [Running Rewrite without build tool plugins](/running-recipes/running-rewrite-without-build-tool-plugins.md)
 * [Running Rewrite on a multi-module Maven project](/running-recipes/multi-module-maven.md)
-* [Popular recipe guides](/running-recipes/popular-recipe-guides/)
+* [Popular recipe guides](/running-recipes/popular-recipe-guides/README.md)
```
Comment (mike-solomon) [inline running-recipes/running-recipes.md:7]: Similar question here - was this link broken?

**Review by mike-solomon — APPROVED** [review]
Thanks for fixing this! Two minor questions below - feel free to merge after taking a look.

File: SUMMARY.md (hunk 1 above)
Comment (timtebeek) [reply SUMMARY.md:1435]: My IDE gave me a warning on this link, and it does appear to be lower case:
https://github.com/openrewrite/rewrite-docs/blob/master/reference/recipes/core-readme.md
On other projects we try not to commit if the IDE has any warning, hence why I changed this now.

File: running-recipes/running-recipes.md (hunk 2 above)
Comment (timtebeek) [reply running-recipes/running-recipes.md:7]: Similarly: IDE warning on a broken link; figured the result is likely the same, but one less warning this way if it's all the same to you.

File: SUMMARY.md (hunk 1 above)
Comment (mike-solomon) [reply SUMMARY.md:1435]: Cool - thanks for taking care of both of those issues then!

**Pull request merged**